### PR TITLE
[Android] Fix animations within transformed hierarchy

### PR DESCRIPTION
**Pull request by davidjohnoliver**
GitHub Issue (If applicable): #

## PR Type
What kind of change does this PR introduce?

Bugfix

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](../README.md#supported)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](doc/articles/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/nventive/Uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

https://nventive.visualstudio.com/Umbrella/_workitems/edit/151547

### CODE REVIEW

File: src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java [hunk 1]
```diff
@@ -851,13 +863,24 @@ protected void removeChildRenderTransform(View child) {
...
+	}
+
 	@Override
 	protected final boolean getChildStaticTransformation(View child, Transformation outTransform) {
 		Matrix renderTransform = _childrenTransformations.get(child);
 		if (renderTransform == null || renderTransform.isIdentity()) {
 			outTransform.clear();
 		} else {
 			outTransform.getMatrix().set(renderTransform);
+//			if (_shouldInvalidate) {
```
Comment (MatFillion) [inline src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java]: any reason for keeping the commented code?

Comment (davidjohnoliver) [reply src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java]: There is an excellent reason... I deleted it in VS, but didn't save, then edited the file in Android Studio and lost the changes in VS

File: src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java [hunk 2]
```diff
@@ -231,13 +234,22 @@ public final void setNativeTextBlockLayout(android.text.Layout layout, int leftP
 		_topTextBlockPadding = topPadding;
 	}
 
+	@Override
+	protected boolean drawChild(Canvas canvas, View child, long drawingTime) {
```
Comment (dr1rrb) [inline src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java]: Is this useful ?

File: src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java (hunk 1 above)
Comment (davidjohnoliver) [reply src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java]: fixed

File: src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java (hunk 2 above)
Comment (davidjohnoliver) [reply src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java]: No, see reply above to Mathieu :D

Comment (dr1rrb) [reply src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java]: Well you removed it while `I was reviewing :)

**Pull request merged**